CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Bug Fixes
   - **Amazon IoT**
     - Fixed a crash in AWSIoTManager when importing PKCS12 data with an incorrect passphrase. (See [#1166](https://github.com/aws-amplify/aws-sdk-ios/issues/1166))
+  - **AWSCognitoIdentityProvider**
```

Comment:
Let's use "AWSMobileClient, Amazon Cognito Identity Provider" rather than the actual class name as the header. Otherwise LGTM.

Reply:
Oops -- sorry, i just saw this prior to the squash-rebase:
Updating the CHANGELOG here:
https://github.com/aws-amplify/aws-sdk-ios/pull/2252